ocrd_tool.md

```diff
@@ -19,6 +19,27 @@ Optionally, workflow processors can be notified that this file is potentially
 large and static (e.g. a fixed dataset or a precomputed model) and should be cached
 indefinitely after download by setting the `cacheable` property to `true`.
 
+The filename itself, i.e. the concrete value `<fpath>` of a file parameter
+should be resolved in the following way:
+
+* If `<fpath>` is an `http`/`https` URL: Download to a temporary directory (if
+  `cacheable==False`) or a semi-temporary cache directory (if `cacheable==True`)
+* If `<fpath>` is an absolute path: Use as-is.
+* If `<fpath>` is a relative path, try resolving the following paths and return
+  the first one found if any, raise a `FileNotFoundError` otherwise:
+  * `$CWD/<fpath>`
+  * If an environment variable is defined that has the name of the processor in
+    upper-case and with `-` replaced with `-` and followed by `_PATH` (e.g. for a processor
+    `ocrd-dummy`, the variable would need to be called `OCRD_DUMMY_PATH`):
+    * Split the variable value at `:` and try to resolve by prepending each token
+      with `<fname>`
+  * `$VIRTUAL_ENV/share/<name-of-processor>/<fpath>`
+  * `$HOME/.local/share/<name-of-processor>/<fpath>`
+  * `$HOME/.config/<name-of-processor>/<fpath>`
+  * `/usr/local/share/<name-of-processor>/<fpath>`
+  * If it is a Python implementation: Replace `/` with `.` in `<fpath>` and try to
```

Comment:
```suggestion
  * If it is a Python implementation: Replace leading `/` with `.` in `<fpath>` and try to
```

Reply:
No, I actually mean `s,/,.,g`. But that's nonsense, `pkg_resources.resource_filename(__name__, "<fpath>")` will handle path separators correctly.

A leading `/` would mean it should be interpreted as an absolute file.